RELEASING.md

```diff
@@ -57,8 +51,11 @@ You'll also need to have permissions to push directly to the `master` branch.
        python3 setup.py sdist  # create a source distribution
        python3 setup.py bdist_wheel  # create a universal wheel
 
+1. Attach the resulting binaries in (`dist/snorkel-x.x.x.*`) to the release.
+
 1. Check that everything looks correct by uploading the package to the PyPI test server:
 
+       pip install twine  # if not installed
        twine upload dist/* -r pypitest  # publish to test.pypi.org
        python3 -m venv test_snorkel  # create a virtualenv for testing
        source test_snorkel/bin/activate  # activate virtualenv
```

Comment:
update RTD instructions: trigger a build for master to pull new tags, go to versions tab and add new tag, go to admin/advanced to set new default version